### PR TITLE
Fix grammar in Using a Service to Expose Your App tutorial

### PR DESCRIPTION
**Pull request by Krishnamohan-Yerrabilli**
### Why:

unwanted articles usage

### What's being changed

Changed "to desired state" to "to the desired state" -> article usage fixed
Changed "via the creation" to "via the creation" -> doesn't have proper article so added it
Added " including selector" to " including a selector" -> article usage problem
Changed "defining selector" to "defining the selector" article usage fixed
Removed "pods in fact have a lifecycle" to "pods have a lifecycle" -> "in fact" may be unnecessary in this sentence, so removed 

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the self-review checklist.

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html [hunk 1]
```diff
@@ -40,7 +40,7 @@ <h3>Overview of Kubernetes Services</h3>
...
 			</ul>
 			<p>More information about the different types of Services can be found in the <a href="/docs/tutorials/services/source-ip/">Using Source IP</a> tutorial. Also see <a href="/docs/concepts/services-networking/connect-applications-service">Connecting Applications with Services</a>.</p>
-			<p>Additionally, note that there are some use cases with Services that involve not defining <code>selector</code> in the spec. A Service created without <code>selector</code> will also not create the corresponding Endpoints object. This allows users to manually map a Service to specific endpoints. Another possibility why there may be no selector is you are strictly using <code>type: ExternalName</code>.</p>
+			<p>Additionally, note that there are some use cases with Services that involve not defining the <code>selector</code> in the spec. A Service created without <code>selector</code> will also not create the corresponding Endpoints object. This allows users to manually map a Service to specific endpoints. Another possibility why there may be no selector is you are strictly using <code>type: ExternalName</code>.</p>
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html:43]: Nit; I'd prefer to fix the fix before we merge it:
```suggestion
			<p>Additionally, note that there are some use cases with Services that involve not defining a <code>selector</code> in the spec. A Service created without <code>selector</code> will also not create the corresponding Endpoints object. This allows users to manually map a Service to specific endpoints. Another possibility why there may be no selector is you are strictly using <code>type: ExternalName</code>.</p>
```

Comment (Krishnamohan-Yerrabilli) [reply content/en/docs/tutorials/kubernetes-basics/expose/expose-intro.html:43]: yaa sure, I will change that @sftim

**Comment by sftim** [comment]
/retitle Fix grammar in Using a Service to Expose Your App tutorial

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Comment by Krishnamohan-Yerrabilli** [comment]
> Thanks
> 
> /lgtm /approve

Pleasure is mine!

**Pull request merged**